lamindb/_transform.py

```diff
@@ -20,23 +20,36 @@ def __init__(transform: Transform, *args, **kwargs):
     if len(args) == len(transform._meta.concrete_fields):
         super(Transform, transform).__init__(*args, **kwargs)
         return None
-    name: str | None = kwargs.pop("name") if "name" in kwargs else None
     key: str | None = kwargs.pop("key") if "key" in kwargs else None
+    description: str | None = (
+        kwargs.pop("description") if "description" in kwargs else None
+    )
     revises: Transform | None = kwargs.pop("revises") if "revises" in kwargs else None
     version: str | None = kwargs.pop("version") if "version" in kwargs else None
     type: TransformType | None = kwargs.pop("type") if "type" in kwargs else "pipeline"
     reference: str | None = kwargs.pop("reference") if "reference" in kwargs else None
     reference_type: str | None = (
         kwargs.pop("reference_type") if "reference_type" in kwargs else None
     )
-    if "is_new_version_of" in kwargs:
-        logger.warning("`is_new_version_of` will be removed soon, please use `revises`")
-        revises = kwargs.pop("is_new_version_of")
+    if "name" in kwargs:
+        if key is None:
+            key = kwargs.pop("name")
+            logger.warning(
+                f"`name` will be removed soon, please pass '{key}' to `key` instead"
+            )
+        else:
+            if description is None:
+                description = kwargs.pop("name")
+                logger.warning(
+                    f"`name` will be removed soon, please pass '{description}' to `description` instead"
+                )
+            else:
+                raise ValueError("name doesn't exist anymore `description`")
```

Comment:
It is unclear to me what this tries to convey.

Reply:
Fixed.